.github/workflows/gitops-ci-openshift.yaml

```diff
@@ -0,0 +1,31 @@
+name: Simulate CI pipeline on Openshift
```

Comment:
Many of our tutorials involve GH action workflows, which is great. It would be nice if we can split up all the GH action workflows we have under the .github/workflows folder into subfolders so that it is clear what belongs to Iter8's own CI and what belongs to Iter8 tutorials.

For starters, it would be nice to simply create .github/workflows/tutorials and move this under that folder.

Reply:
Make sense. Let's gradually make this change, starting from this PR.